Header head
int32 num_vertex
float64 risk
float64 cost
float64 goal_x_cov
float64 goal_y_cov
float64 goal_xy_cov